### PR TITLE
Update tr.json

### PR DESCRIPTION
**Pull request by Melocet**
Turkish Translation by Melocet#1645

### CODE REVIEW

**Comment by Melocet** [comment]
Turkish Translation by Melocet#1645

**Comment by Melocet** [comment]
Typo errors fixed
All lines rechecked

File: tr.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "live" : true,
     "translations" : {
         "meta" : {
-            "thForDate" : false,
+            "thForDate" : true,
```
Comment (Krymonota) [inline tr.json:7]: Just to be sure: This change causes "st", "nd" and "th" to be appended to the date. Does this really apply to Turkish?

Comment (Melocet) [reply tr.json]: No, soory its fixed.

**Pull request merged**